MNE/mne/mne_bem.h

```diff
@@ -164,11 +164,63 @@ class MNESHARED_EXPORT MNEBem
 
```

Comment:
Please add the following convenient functions 
void clear();
inline bool isEmpty() const;

See MNESourceSpace for more information